Multisig.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -267,8 +258,8 @@
         "repositoryURL": "https://github.com/WalletConnect/WalletConnectSwift.git",
         "state": {
           "branch": null,
-          "revision": "f951da68a32666712e8c80ba0c77e9df5dce9fd6",
-          "version": "1.6.0"
+          "revision": "f316e6dd91639d6e3e5a4c8860d823c08cc71bc3",
+          "version": "1.6.2"
```

Comment:
@liliya-soroka please pay attention to basic flows with WalletConnect after this PR will be merged. There were some changes in the lib.

Reply:
hm.. this might break stuff actually :(

Reply:
can we revert back to 1.6.0?